pkg/debuginfo/store.go

```diff
@@ -63,6 +63,8 @@ type CacheConfig struct {
 type Store struct {
 	debuginfopb.UnimplementedDebugInfoServiceServer
 
+	debuginfodClientCache *ObjectStorageDebugInfodClientCache
```

Comment:
```suggestion
	debuginfodClientCache DebugInfodClient
```
I think it's better to use the interface here.